values.tf

```diff
@@ -1,28 +1,20 @@
 locals {
-  helm_values = [{
+  helm_values = [merge({}, var.oidc == null ? null : {
     argo-cd = {
       configs = {
         secret = {
           extra = {
-            "oidc.default.clientSecret" = "${replace(local.oidc.client_secret, "\\\"", "\"")}"
+            "oidc.default.clientSecret" = "${replace(var.oidc.clientSecret, "\\\"", "\"")}"
```

Comment:
I know this was already the case before, but I’m not sure to understand why we put the client secret in Argo CD secret if we pass it in clear in Argo CD config map. IIRC, we should use a secret reference in OIDC config.

Reply:
Fixed